api/delete-histogram.md

```diff
@@ -18,15 +32,26 @@ DELETE
 
  * `uuid` The UUID of the check to which the metric belongs.
  * `metric` The name of the metric from which to delete data.
+ * `metric_pattern_including_wildcards` : A metric naming pattern string including wildcards.
+ * `query` : See [Tag Queries](/tag_queries.md) for more info on tag queries.
 
 ### Headers
 
- * `X-Snowth-Delete-Time: <end>` (required)
-   * `end` The end timestamp for the delete operation. All data from before this specified time is deleted. Time is represented in seconds since the epoch.
- * `X-Snowth-Full-Delete: <value>` (optional)
```

Comment:
So there is no longer an option to journal deletes, and all operations must be repeated against all cluster nodes?

Reply:
single metric deletes can still be journaled IF you specify this in the header (DELETE nnt/, text/, histogram/), this has not changed.
however, all of the multiple metric deletes (wildcard, tag query and check) all require confirmation and will not journal, in order to be cautious on development, testing and in production.
Riley requested confirmation in the original JIRA, so that the user can hopefully avoid the "rm -rf /" mistake in the first place by first looking at the response when the command is not confirmed, and you get back a list of what will actually be deleted if you reissue with confirmation.  In the same way, being cautious with journaling a large delete is playing it safe (for now).
I didn't want to enable journaling without adding another series of tests and things, so if customers or us want to journal these kinds of deletes, then we would be certain that it would work correctly.  And it's not too hard to script a curl command loop that would do the deletes against all nodes in the cluster.
So I figured let's get this out and see if customers really want and need the journaling of multi-metric deletes before we go through the trouble to enable it and validate it (and warn users that if you journal the multi-metric delete then you will not be able to stop it without killing all the nodes, and you won't be able to reconstitute the deleted data).

Reply:
Understood, thanks